[0001].[2021.04.03.02.03.19.638076].SRV1.INF:Main.py.<module>.319:Main Program Started
[0002].[2021.04.03.02.03.19.638576].SRV1.ERR:Main.py.<module>.322:Artificial Error Message for testing purpose
[0003].[2021.04.03.02.03.19.639076].SRV1.INF:Main.py.<module>.323:Command Line iniFileName=default.ini
[0004].[2021.04.03.02.03.19.639577].SRV1.INF:Main.py.<module>.325:Command Line VerboseMode=None
[0005].[2021.04.03.02.03.19.640077].SRV1.INF:Main.py.<module>.328:AppSpoManager.DebugVerboseModeOn=True
[0006].[2021.04.03.02.03.19.640577].SRV1.INF:Main.py.<module>.329:AppSpoManager.DebugInfoMessages=True
[0007].[2021.04.03.02.03.19.641078].SRV1.INF:Main.py.<module>.330:AppSpoManager.DebugErrorMessages=True
[0008].[2021.04.03.02.03.19.641578].SRV1.INF:Main.py.<module>.331:AppSpoManager.DebugTargetConsole=False
[0009].[2021.04.03.02.03.19.642078].SRV1.INF:Main.py.<module>.332:AppSpoManager.DebugTargetFile=True
[0010].[2021.04.03.02.03.19.642579].SRV1.INF:Main.py.<module>.333:AppSpoManager.DebugTargetFilePath=Debugs\Debug1
[0011].[2021.04.03.02.03.19.643079].SRV1.INF:Main.py.<module>.334:AppSpoManager.DebugLinesPerFile=1000
[0012].[2021.04.03.02.03.19.643579].SRV1.INF:Main.py.<module>.335:AppSpoManager.AppRunFolder=c:\Apps\PythonTry1
[0013].[2021.04.03.02.03.19.644080].SRV1.INF:Main.py.<module>.358:ModuleFileNameOnly=Main.py
[0014].[2021.04.03.02.03.19.644580].SRV1.INF:Main.py.<module>.366:File=c:\Apps\PythonTry1\Main.py
[0015].[2021.04.03.02.03.19.645080].SRV1.INF:Main.py.<module>.367:FileDirName=c:\Apps\PythonTry1
[0016].[2021.04.03.02.03.19.645581].SRV1.INF:Main.py.<module>.368:FileAbsPath=c:\Apps\PythonTry1\Main.py
[0017].[2021.04.03.02.03.19.646081].SRV1.INF:Main.py.<module>.371:ModuleName1=__main__
[0018].[2021.04.03.02.03.19.646581].SRV1.INF:Main.py.<module>.372:ModuleName2=Main
[0019].[2021.04.03.02.03.19.647082].SRV1.INF:Main.py.<module>.381:Main Program Started
[0020].[2021.04.03.02.03.19.647582].SRV1.INF:Main.py.<module>.382:Python Executable Path=C:\Program Files\Python39\python.exe
[0021].[2021.04.03.02.03.19.648082].SRV1.INF:Main.py.<module>.383:Python Interpreter Version=3.9.2 (tags/v3.9.2:1a79785, Feb 19 2021, 13:44:55) [MSC v.1928 64 bit (AMD64)]
[0022].[2021.04.03.02.03.19.648583].SRV1.INF:Main.py.<module>.386:Sys.Path=['c:\\Apps\\PythonTry1', 'C:\\Program Files\\Python39\\python39.zip', 'C:\\Program Files\\Python39\\DLLs', 'C:\\Program Files\\Python39\\lib', 'C:\\Program Files\\Python39', 'C:\\Users\\user2\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\Program Files\\Python39\\lib\\site-packages']
[0023].[2021.04.03.02.03.19.680102].SRV1.INF:Main.py.<module>.433:Showing MainWindow
[0024].[2021.04.03.02.03.21.464902].SRV1.INF:Main.py.actionDebugTest1Handler.154:192.168.0.1, KAT1.ODA1, KORÝDOR DUVAR
192.168.0.2, KAT2.ODA2, TAVAN HOPARLÖR
192.168.0.3, KAT3.ODA3, KAPI_ÖNÜ_ANANS
[0025].[2021.04.03.02.03.21.465903].SRV1.INF:Main.py.actionDebugTest1Handler.169:Node[0] Order=[0] IP=192.168.0.1] Place=]KAT1.ODA1] Desc=[KORÝDOR DUVAR
[0026].[2021.04.03.02.03.21.466903].SRV1.INF:Main.py.actionDebugTest1Handler.169:Node[1] Order=[1] IP=192.168.0.2] Place=]KAT2.ODA2] Desc=[TAVAN HOPARLÖR
[0027].[2021.04.03.02.03.21.467904].SRV1.INF:Main.py.actionDebugTest1Handler.169:Node[2] Order=[2] IP=192.168.0.3] Place=]KAT3.ODA3] Desc=[KAPI_ÖNÜ_ANANS
